app/models/answer.rb

```diff
@@ -0,0 +1,17 @@
+class Answer < ActiveRecord::Base
+
+has_many :comments, as: :commentable, dependent: :destroy
+has_many :votes, as: :votable, dependent: :destroy
+
+belongs_to :user
+belongs_to :question
+
+validates :content, :user_id, :question_id, presence: true
```

Comment:
Better to validate question rather than question_id - ensures the id is present and points to a valid object. Validating id would let you save a record with question_id = -23 for example.